styles.css

```diff
@@ -409,6 +409,10 @@ a {
   margin: 3px;
 }
 
+.nav_menu2 a:hover {
+  color: #f55800;
+}
+
```

Comment:
Nice work on implementing the functionalities of the `nav` 👍👍 Kindly note that your project looks a bit different from the given design😉 Kindly see the comparison below and take note of the colors and implement the necessary change😉 as we are required to stick to the design in this milestone👍

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="289" alt="yours" src="https://user-images.githubusercontent.com/64643995/232853380-fd215c87-7a9e-47c3-9273-dd7d7eb2fcd7.png">|<img width="292" alt="design menu" src="https://user-images.githubusercontent.com/64643995/232853461-4dceeebb-db6d-4e6c-bf35-6ceb542438ca.png">|

Reply:
Hello @Ibizugbe Thanks for your review but i am not sure i get you correctly. Dont you think the difference is from our different screen? I have checked the colors in figma and its the same with mine.